BEIMA.Client/cypress/integration/LoginPage.spec.js

```diff
@@ -11,24 +13,8 @@ describe('Login Page', () => {
     cy.get('.invalid-feedback').its('length').should('eq',2)
   })
 
-  it('Errors dont appear until touched', () => {
-    cy.visit('http://localhost:3000/login')
-    cy.get('.invalid-feedback').its('length').should('eq',2)
-    
-    cy.get('.invalid-feedback').each((val) => {
-      cy.wrap(val).should('not.be.visible')
-    })
-    cy.get('[id=submitBtn]').click()
-    cy.get('.invalid-feedback').each((val) => {
-      cy.wrap(val).should('be.visible')
-    })
-    cy.wait(1250)
-    cy.get('[id=submitBtn]').should('be.disabled')
-  })
-  
-
```

Comment:
I think due to the submit form change, we don't get the error feedback anymore, so this test fails. Instead, we get this, which I think is still fine. It also shows if the username is entered but no password is and the user tries to submit
![image](https://user-images.githubusercontent.com/58532471/164091001-4f676657-084d-4ce8-91d8-d9284a76d731.png)